src/sprites.py

```diff
@@ -174,14 +220,14 @@ def __init__(self, pos: Coordinate, frames, groups, collision_sprites: pygame.sp
         self.tool_active = False
         self.just_used_tool = False
         self.apply_tool = apply_tool
-        self.pause_menu = pause_menu(self.font)
-        self.settings_menu = settings_menu(self.font, self.sounds)
-        # seeds 
+        self.pause_menu = pause_menu.PauseMenu(self.font)
+        self.settings_menu = settings_menu.SettingsMenu(self.font, self.sounds)
+        # seeds
```

Comment:
I'm not sure about enforcing these sort of qualifications -- it doesn't really add anything and requires extra repetition + verbosity...

Are there ways to turn this particular lint off? It can still make sense for external library symbols, but maybe not our own classes :thinking:

Reply:
This one is from me, not the linter, to be honest. I can switch this to import the class instead of the module. I don't have a strong opinion about it, even I prefer to import only modules. :)

Reply:
I think often it's fine indeed, e.g. this other change:
```py
            'animations': support.animation_importer('images', 'animations'),
            'soil': support.import_folder_dict('images/soil'),
            'soil water': support.import_folder_dict('images/soil water'),
            'tomato': support.import_folder('images/plants/tomato'),
            'corn': support.import_folder('images/plants/corn'),
            'rain drops': support.import_folder('images/rain/drops'),
            'rain floor': support.import_folder('images/rain/floor'),
            'objects': support.import_folder_dict('images/objects')
```

Just in cases like here, it's more repetition. But it's not very important either way :wink: so don't spend much time fixing it.